docs/add-new-token.md

```diff
@@ -0,0 +1,132 @@
+# Adding a New Token to the List
+
+The goal of this guide is to walk you through the steps needed to add a new token to the token list.
+
+## Before you begin
+
+You need to have some information about the token you want to add to the list before you begin this process:
+
+1. Chain ID
+
+The new token can be added to testnet and/or mainnet and the chain ID for each is the following:
+
+- Testnet: 743111
+- Mainnet: 43111
+
+> If you want to add the token to both testnet and mainnet you need to execute these steps for each chain (one for testnet and another for mainnet).
+
+1. Token Address
```

Comment:
```suggestion
2. Token Address
```


This is how it looks

<img width="1132" alt="image" src="https://github.com/user-attachments/assets/2b6ac04c-64a1-4592-9805-13f3b2920419" />


it seems the numbering is being lost

Reply:
I added `2.` first time, but got this eslint error:
>  86:2  error  Ordered list item prefix: Expected: 1; Actual: 2; Style: 1/1/1  markdownlint/md029

Reply:
replace it by sub-title